src/application/extension-popup/app.tsx

```diff
@@ -0,0 +1,45 @@
+import { Route, Routes } from 'react-router';
+
+import { EXTENSION_POPUP_ROUTE, useExtensionPopup } from 'shared/extension';
+import { Closable } from 'shared/ui';
+
+import { Footer, TopBar } from './components';
+import { PopupHomeView } from './popup-home-view';
+import { CustomizationSettings, MainSettingsMenu } from './settings-view';
+
```

Comment:
let's apply group these under `./views` and you can remove `-view` suffix, also inside each of them you can rename them to `CustomizationSettingsView`, `MainSettingsView` and `HomeView` instead of `App` + renaming in export